backend/Ript-Fitness-Backend/src/main/java/com/riptFitness/Ript_Fitness_Backend/domain/model/UserProfile.java

```diff
@@ -7,7 +7,9 @@
 import jakarta.persistence.GeneratedValue;
 import jakarta.persistence.GenerationType;
 import jakarta.persistence.Id;
+import jakarta.persistence.OneToMany;
 import jakarta.persistence.ElementCollection;
+import jakarta.persistence.CascadeType;
```

Comment:
No other changes in the rest of this file? Why does it need these two new imports?